panel/reactive.py

```diff
@@ -1284,21 +1281,14 @@ def _convert_column(
         if dtype.kind == 'M':
             if values.dtype.kind in 'if':
                 if getattr(dtype, 'tz', None):
-                    # dtype has a timezone
-                    if dtype.tz == dt.timezone.utc:
-                        # Milliseconds to nanoseconds, to datetime64.
-                        converted = (values * 1e6).astype('datetime64[ns]')
-                    else:
-                        import pandas as pd
-
-                        # Using pandas to convert from milliseconds
-                        # timezone-aware, to UTC nanoseconds, to datetime64.
-                        converted = (
-                            pd.Series(pd.to_datetime(values, unit="ms"))
-                            .dt.tz_localize(dtype.tz)
-                            .dt.tz_convert('utc')
-                            .dt.tz_localize(None)
-                        )
+                    import pandas as pd
```

Comment:
@philippjfr commented on https://github.com/holoviz/panel/pull/6879 about the original code:

```
converted = (
    pd.Series(pd.to_datetime(values, unit="ms"))
    .dt.tz_localize(dtype.tz)
    .dt.tz_convert('utc')
    .dt.tz_localize(None)
)
````

> What is the intention behind these last two lines? This will soon cause pandas to error and also I'm quite confused. This localizes in the original provided timezone, then shifts it to UTC, which seems wrong.

I think that was me just being confused with timezone handling and Bokeh serialization to Unix timestamps. It seems to work fine with these changes!

Reply:
Perfect, thanks for confirming!